src/main/java/seedu/contax/commons/core/Messages.java

```diff
@@ -18,5 +18,5 @@
     public static final String MESSAGE_INVALID_DATE = "The date provided is invalid!";
     public static final String MESSAGE_INVALID_TIME = "The time provided is invalid!";
 
-    public static final String MESSAGE_INVALID_INPUT = "=/ provided is invalid";
+    public static final String MESSAGE_INVALID_BATCH_INPUT = "the field of batch command provided is invalid \n%1$s";
```

Comment:
You should use the standardized message for this (i.e. `MESSAGE_INVALID_COMAND_FORMAT`).

Reply:
thanks removed